msteams-platform/samples/virtual-table-entity-references.md

```diff
@@ -0,0 +1,292 @@
+|---|---|---|
+|`m365_audio` |StringType |Audio metadata, if the item is an audio file. Read-only. Only on OneDrive Personal. This attribute is a JSON encoded string. { "album": "string",  "albumArtist": "string", artist": "string", bitrate": 128, "composers": "string", copyright": "string", "disc": 0, "discCount": 0, "duration": 567, "genre": "string", "hasDrm": false, "isVariableBitrate": false, "title": "string", "track": 1, "trackCount": 16, "year": 2014 }|
+|`M365_bundle` |StringType |Bundle metadata, if the item is a bundle. Read-only. This attribute is a JSON encoded string. For example { "childCount": 3, "album": { "@odata.type": "microsoft.graph.album" }, } |
+|`m365_collaborationrootid` |StringType |Collaboration root ID of the collaboration session the record is associated with. If the record is associated with multiple collaboration sessions this will be returned as a comma delimited string. Note that this attribute won't be returned when retrieving multiple records. |
+|`m365_copy` |StringType |If present in the request then a copy operation is performed. |
+|`m365_createdby` |StringType |Identity of the user, device, and application, which created the item. Read-only. This attribute is a JSON encoded string. For example {"user":{"displayName":"User Name","email":"alias@contoso.com","ID":"a298b975-3493-4d9e-b2d4-3cad78f00000"},"group": null,"application","device" } |
+|`m365_createddatetime` |DateTimeType |Date and time of item creation. Read-only. |
+|`m365_ctag` |StringType |An eTag for the content of the item. This eTag isn't changed if only the metadata is changed. Note. This property isn't returned if the item is a folder. Read-only. |
+|`m365_deleted` |StringType |Information about the deleted state of the item. Read-only. This attribute is a JSON encoded string. For example { "state": "string" } |
+|`m365_description` |StringType |Provides a user-visible description of the item. Read-write. Only on OneDrive Personal. |
+|`m365_driveid` |StringType |The Identifier for the drive that contains the drive item.|
+|`m365_etag` |StringType |eTag for the entire item (metadata + content). Read-only. |
+| `m365_file` |StringType |File metadata, if the item is a file. Read-only. This attribute is a JSON encoded string. For example {"hashes":{"crc32Hash","quickXorHash":"Biuzvwdu+Tmu6yRefayD27hD9vD=","sha1Hash","sha256Hash" },"mimeType":"application/vnd.openxmlformats-officedocument.wordprocessingml.document","processingMetadata" } |
+| `m365_filesysteminfo` |StringType |File system information on client. This attribute is a JSON encoded string. For example {"createdDateTime":"2022-07-21T15:02:47+00:00","lastAccessedDateTime","lastModifiedDateTime":"2022-07-21T15:02:55+00:00"} |
+|`m365_folder` |StringType | Folder metadata, if the item is a folder. Read-only. This attribute is a JSON encoded string. For example {"childCount":0,"view" } |
+|`m365_graphdriveitemid` |GuID |Unique Identifier of the graph drive item. |
```

Comment:
Change to

| `m365_graphdriveitemId` | guid | Unique Identifier of the graph drive item. |